client/my-sites/plans-v2/utils.ts

```diff
@@ -209,6 +210,7 @@ export function itemToSelectorProduct(
 			productSlug: item.product_slug,
 			iconSlug: `${ item.product_slug }_v2`,
 			displayName: getJetpackProductDisplayName( item ),
+			shortName: getJetpackProductShortName( item ),
```

Comment:
The type above adds `shortName` as required, but it is only implemented on products, not plans. This will cause TS errors and should be fixed, even with an empty string if no such data exists for plans.

Reply:
Good catch.